src/dataframe/dataframe.jl

```diff
@@ -234,7 +236,9 @@ function Base.getindex(df::DataFrame, col_ind::ColumnIndex)
 end
 
 # df[MultiColumnIndex] => (Sub)?DataFrame
-function Base.getindex{T <: ColumnIndex}(df::DataFrame, col_inds::AbstractVector{T})
+function Base.getindex{T <: ColumnIndex}(df::DataFrame,
+                                         col_inds::Union{AbstractVector{T},
+                                                         AbstractVector{Nullable{T}}})
```

Comment:
I'm guessing there are other operations that somehow rely on the fact that indexing with a Nullable works? Otherwise, this seems a little overly-permissive and could just be `AbstractVector{T}`.